apps/src/signUpFlow/FinishStudentAccount.tsx

```diff
@@ -0,0 +1,199 @@
+  const [parentEmail, setParentEmail] = useState('');
+  const [parentEmailOptInChecked, setParentEmailOptInChecked] = useState(false);
+  const [name, setName] = useState('');
+  const [age, setAge] = useState('');
+  const [state, setState] = useState('');
+  const [gender, setGender] = useState('');
+
+  const onIsParentChange = (): void => {
+    const newIsParentCheckedChoice = !isParent;
+    setIsParent(newIsParentCheckedChoice);
+    sessionStorage.setItem(
+      IS_PARENT_SESSION_KEY,
+      `${newIsParentCheckedChoice}`
+    );
+  };
+
+  const onParentEmailChange = (
+    e: React.ChangeEvent<HTMLInputElement>
+  ): void => {
+    const newParentEmail = e.target.value;
+    setParentEmail(newParentEmail);
+    sessionStorage.setItem(PARENT_EMAIL_SESSION_KEY, newParentEmail);
+  };
+
+  const onParentEmailOptInChange = (): void => {
+    const newParentEmailOptInCheckedChoice = !parentEmailOptInChecked;
+    setParentEmailOptInChecked(newParentEmailOptInCheckedChoice);
+    sessionStorage.setItem(
+      PARENT_EMAIL_OPT_IN_SESSION_KEY,
+      `${newParentEmailOptInCheckedChoice}`
+    );
+  };
+
+  const onNameChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const newName = e.target.value;
+    setName(newName);
+    sessionStorage.setItem(DISPLAY_NAME_SESSION_KEY, newName);
+  };
+
+  const onAgeChange = (e: React.ChangeEvent<HTMLSelectElement>): void => {
+    const newAge = e.target.value;
+    setAge(newAge);
+    sessionStorage.setItem(USER_AGE_SESSION_KEY, newAge);
+  };
+
+  const onStateChange = (e: React.ChangeEvent<HTMLSelectElement>): void => {
+    const newState = e.target.value;
+    setState(newState);
+    sessionStorage.setItem(USER_STATE_SESSION_KEY, newState);
+  };
+
+  const onGenderChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const newGender = e.target.value;
+    setGender(newGender);
+    sessionStorage.setItem(USER_GENDER_SESSION_KEY, newGender);
+  };
+
+  return (
+    <div className={style.finishAccountContainer}>
+      <div className={style.headerTextContainer}>
+        <Heading2>{locale.finish_creating_student_account()}</Heading2>
+        <BodyTwoText>{locale.tailor_experience()}</BodyTwoText>
+      </div>
+      <div className={style.inputContainer}>
+        <div className={style.parentInfoContainer}>
+          <span className={style.parentCheckboxContainer}>
+            <div className={style.parentCheckbox}>
+              <Checkbox
+                label=""
+                checked={isParent}
+                onChange={onIsParentChange}
+                name="isParent"
+              />
+            </div>
+            <BodyTwoText>{locale.i_am_a_parent_or_guardian()}</BodyTwoText>
+          </span>
+          {isParent && (
+            <>
+              <BodyThreeText className={style.studentQuestionLabel}>
+                <strong>{locale.parent_guardian_email()}</strong>
+              </BodyThreeText>
```

Comment:
You can wrap all of the labels in a `<label>` element and connect it to their respective text field elements so when you click on the label it'll select the text field (also improves accessibility!). The `for` attribute connects to the `name` attribute on the input fields. 

```suggestion
            <label for="parentEmail">
              <BodyThreeText className={style.studentQuestionLabel}>
                <strong>{locale.parent_guardian_email()}</strong>
              </BodyThreeText>
            </label>
```

Reply:
On it!